libs/@guardian/source-react-components-development-kitchen/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"extends": "../../../tsconfig.base.json",
 	"files": [],
-	"include": [".", "../../../@types/window.d.ts"],
+	"include": [".", "../../../@types/window.d.ts", ".storybook/*"],
```

Comment:
Do we need to add this here?

Reply:
without these, the hidden ".storybook" directory won't be captured by tsconfig as it'll ignore hidden files/dirs by default afaik

Reply:
yeah i get that, i just mean if we weren't checking the storybook config files previously, do we need to start doing it now?

Reply:
Is there any actual types to check within the config?

Reply:
ah good point - I think it's worth doing type checking against them.. can't think of a reason why we wouldn't. We also have types here: https://github.com/guardian/csnx/pull/220/files#diff-a03d3fafe3aae070b0e2c3e77cfb2f949ad227868d805467765a6299df112636R45 that @tjsilver added in one of our earlier preparation PRs to move Source in

Reply:
ok if you think it makes sense then sure, just good to question the motivations if we're changing things 👍